src/api/comments.ts

```diff
@@ -0,0 +1,21 @@
+import { request } from './api';
+
+export const getPostComments = (postId: number): Promise<Comment[]> => {
+  return request(`/comments?postId=${+postId}`);
```

Comment:
postId already a number (all similar places)
```suggestion
export const getPostComments = (postId: number): Promise<Comment[]> => {
  return request(`/comments?postId=${postId}`);
```